draft-fossati-tls-attestation.md

```diff
@@ -154,74 +154,108 @@ Several formats for encoding evidence are available, such as:
-in  the TLS handshake, such that the details about the attestation technology are
-agnostic to the TLS handshake itself. Later versions of the specification will 
-support the passport model as well. 
+This specification defines how to support the background check model in the TLS
+handshake, such that the details about the attestation technology are agnostic
+to the TLS handshake itself. Similarly, support for the passport model is
+defined such that details about the attestation results encodings and trust
+relationships are agnostic to the TLS handshake.
+
+To give the peer information that the handshake signing key is properly secured,
+the associated attestation result has to be appraised by the peer. This must be
+the case when either of the two remote attestation topologies is used. Hence,
+attestation evidence about the security state of the signing key is needed,
+which is typically associated with evidence about the overall platform state.
+The platform attestation service ensures that the key attestation service has
+not been tampered with. The platform attestation service issues the Platform
+Attestation Token (PAT) and the key attestation service issues the Key
+Attestation Token (KAT). The security of the protocol critically depends on the
+verifiable binding between these two logically separate units of evidence.
+
+This document does not define how different attestation technologies are
+encoded. This is accomplished by companion specifications.
 
-To give the peer information that the handshake signing key is properly secured, 
-the associated evidence has to be verified by that peer.
-Hence, attestation evidence about the security state of the signing key is needed, which
-is typically associated with evidence about the overall platform state. The platform 
-attestation service ensures that the key attestation service has not been tampered with.
-The platform attestation service issues the Platform Attestation Token (PAT) and the
-key attestation service issues the Key Attestation Token (KAT). The security of the 
-protocol critically depends on the verifiable binding between these two logically separate
-units of evidence.
+# Conventions and Terminology
 
-This document does not define how different attestation technologies are encoded.
-This is accomplished by companion specifications.
+The following terms are used in this document:
 
-# Conventions and Terminology
+{: vspace="0"}
+
+TLS Identity Key (TIK): : A cryptographic key used by one of the peers to
+authenticate itself during the TLS handshake.
+
+The reader is assumed to be familiar with the vocabulary and concepts defined in
+{{-rats-arch}}, and those in {{-rats-kat}}.
+
+"Remote attestation evidence" is more succintly referred to as "evidence", and
+"remote attestation results" is more succintly referred to as "results"
+throughout this document. "Remote attestation credentials", or "credentials", is
```

Comment:
Similar to above, I would suggest "attestation credentials" rather than "credentials" alone.